gui/mozregui/pref_editor.py

```diff
@@ -30,13 +30,13 @@ def data(self, index, role=Qt.DisplayRole):
             if index.column() == 0:
                 return name
             else:
-                return repr(value)
+                return QString(value).__str__()
```

Comment:
Using repr(value) doesn't render the non-ascii characters properly. They are rendered as '\xYZ' characters.